proto/config.proto

```diff
@@ -372,18 +372,26 @@ message TiltOptions
 	optional bool enabled = 1;
 
 	optional int32 tilt1Pin = 2;
-	optional int32 tilt2Pin = 3;
-	optional int32 tiltFunctionPin = 4;
-	optional int32 tiltLeftAnalogUpPin = 5;
-	optional int32 tiltLeftAnalogDownPin = 6;
-	optional int32 tiltLeftAnalogLeftPin = 7;
-	optional int32 tiltLeftAnalogRightPin = 8;
-	optional int32 tiltRightAnalogUpPin = 9;
-	optional int32 tiltRightAnalogDownPin = 10;
-	optional int32 tiltRightAnalogLeftPin = 11;
-	optional int32 tiltRightAnalogRightPin = 12;
-
-	optional SOCDMode tiltSOCDMode = 13;
+	optional int32 factorTilt1LeftX =  3;
```

Comment:
You should never change the tag numbers of existing protobuf fields, this breaks backwards compatibility. You can add as many new fields as you like, and you can change the order of the fields in a struct as long as the tag numbers of existing fields do not change.